relay-redis/src/config.rs

```diff
@@ -1,5 +1,25 @@
 use serde::{Deserialize, Serialize};
 
+const fn default_max_connections() -> u32 {
+    24
+}
+
+/// Additional configuration options for a redis client.
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct RedisConfigOptions {
+    /// Maximum number of connections managed by the pool.
+    #[serde(default = "default_max_connections")]
```

Comment:
nit: You could put `#serde(default)` on top of `RedisConfigOptions`, then any missing value would be derived from the `Default` impl below:

https://serde.rs/container-attrs.html#default

See for example https://github.com/getsentry/relay/blob/25fee3201b1142600487040021875efc120405e8/relay-config/src/config.rs#L657-L659.